src/app/Main/Body/QueryPage/Results/Result-Content.component.tsx

```diff
@@ -0,0 +1,56 @@
+/** @jsx jsx */
+import { css, jsx } from "@emotion/core";
+import { useContext } from "react";
+import { ResultsTableComponent as ResultsTable } from "./Results-Table.component";
+import { ExplanationTextComponent as ExplanationText } from "../../../../Shared/ExplanationText.component";
+import { TableContext } from "../../../../Shared/Context/TableContext";
+
+const dataTableStyle = css`
+    overflow: auto;
+`;
+
+interface ContentInterface {
+    displayRowCol: {
+        isCol: boolean;
+        isRow: boolean;
+    };
+    columnAttributes: string[];
+    getSize: (
+        node: HTMLElement | null,
+        key: "height" | "totalWidth" | "partWidth"
+    ) => void;
+}
+
+export function TableContentComponent(props: ContentInterface): JSX.Element {
+    const { table } = useContext(TableContext);
+
+    const noRowAndCol = !!(

```

Comment:
reject: remove !!, all you compare here are booleans, so no boolean conversion hack necessary.
Try to avoid such statements